src/Themes/ui/Theme.tsx

```diff
@@ -251,7 +260,7 @@ export function refreshTheme(): void {
             color: Settings.theme.primary,
             backgroundColor: Settings.theme.well,
             borderRadius: 0,
-            border: "2px solid white",
+            border: "2px solid " + Settings.theme.welllight,
```

Comment:
IMO this should be theme.white

Reply:
I spent a bit of time mulling this one over.  I went with `welllight` because it's both consistent with other borders in the game and can be made to work well with light-background themes.

Reply:
Although by that logic I probably should have found something else to use above for MuiInput's borderBottomColor.  Hrm.

Reply:
I'm still unfamiliar with Github's code review tools and I'm unsure why `welllight` still appears above, but the "files changed" tab of this PR shows `white`, so I'm calling this done.